examples/seismic/tutorials/13_LSRTM_acoustic.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>Can you double check that section. Some part of the text seem to be missing or werirdly ordered. For example at the begging you have</p><pre class="ql-syntax" spellcheck="false">into the constant-density acoustic wave equation, considering&nbsp;

</pre><p>that is missing the beginning of the sentence.</p><p> </p><ul><li><code>q</code>  is usually for the source, use <code>v</code>  or <code>u_r</code>  for the adjoint wavefield</li><li>Developing =&gt; expanding</li><li>	"Combining the Born's approximation into equation (2). That's not actually correct. You are substituting your equation 3a into 2 to cancel out the background wavefield to get equation 3b.</li><li>Concerning eq 5, put ref to tutorial that derives it.</li><li>For equation (6) this is not true/incomplete. Least square migration is a least-square problem <code>min_m ||J*dm - \delta d||</code>  which gradient is equation 5 and <code>J</code>  is the Jacobian or Born modeling operator</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thank you, @mloubout. I will fix it.